code/client/auth.dm

```diff
@@ -0,0 +1,49 @@
+// in order of execution
+var/list/datum/client_auth_gate/pre_auth_gates = list(
+	new /datum/client_auth_gate/telnet,
+	new /datum/client_auth_gate/guest,
```

Comment:
Just a reminder that we'll need to support Guest accounts if byond continues being down.

See: anyone without a prior account, or who isn't in the connected ckey cache thing byond has.